packages/docs/components/DocsExample.vue

```diff
@@ -1,6 +1,6 @@
 <template>
```

Comment:
`drag-start` -> `dragStart`
Same for all other 2 word events.
We want to keep naming consistent to allow easy search in IDE.

Reply:
But we have kebab-case in code, should we have kebab in api config and locales also?